src/components/DateInput/index.tsx

```diff
@@ -36,13 +36,11 @@ const StyledWrapper = styled.div`
   }
   .calendar {
     font-family: 'Asap';
-    border-color: ${({ theme: { colorsDeprecated } }) =>
-      colorsDeprecated.gray300};
+    border-color: ${({ theme }) => theme.colors.neutral.border};
 
     ${PREFIX}__header {
-      color: ${({ theme: { colorsDeprecated } }) => colorsDeprecated.gray700};
-      background-color: ${({ theme: { colorsDeprecated } }) =>
-        colorsDeprecated.white};
+      color: ${({ theme }) => theme.colors.neutral.textStrong};
```

Comment:
```suggestion
      color: ${({ theme }) => theme.colors.neutral.text};
```

`textStrong` is white as background is dark the text should be light